actix-pgsql-simple/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2018"
 
 [dependencies]
-casbin = "0.9"
+casbin = "1.1"
```

Comment:
Maybe using `casbin = { version = "1.1.0", default-features = false, features = ["incremental"] }` is better

Reply:
`error: aborting due to 24 previous errors`

On my machine, it does not compile properly, it seems that this is some kind of failure of Casbin itself, so I decided not to make any changes.

Reply:
![some errors](https://user-images.githubusercontent.com/36896360/87494816-8bb27d00-c682-11ea-8d66-e791a95b7826.png)

I don’t think these errors are caused by examples, so it may need to be fixed.